content/help/ops/security/debugging-authorization/index.md

```diff
@@ -0,0 +1,238 @@
+the debug output is generated for these policies.
+
+    Check Pilot log with the following command:
+
+    {{< text bash >}}
+    $ kubectl logs $(kubectl -n istio-system get pods -l istio=pilot -o jsonpath='{.items[0].metadata.name}') -c discovery -n istio-system | grep rbac
+    ... ...
+    2018-07-26T22:25:41.009838Z debug rbac building filter config for {sleep.foo.svc.cluster.local map[app:sleep pod-template-hash:3326367878] map[destination.name:sleep destination.namespace:foo destination.user:default]}
+    2018-07-26T22:25:41.009915Z info  rbac no service role in namespace foo
+    2018-07-26T22:25:41.009957Z info  rbac no service role binding in namespace foo
+    2018-07-26T22:25:41.010000Z debug rbac generated filter config: { }
+    2018-07-26T22:25:41.010114Z info  rbac built filter config for sleep.foo.svc.cluster.local
+    2018-07-26T22:25:41.182400Z debug rbac building filter config for {productpage.default.svc.cluster.local map[pod-template-hash:2600844901 version:v1 app:productpage] map[destination.name:productpage destination.namespace:default destination.user:bookinfo-productpage]}
+    2018-07-26T22:25:41.183131Z debug rbac checking role app2-grpc-viewer
+    2018-07-26T22:25:41.183214Z debug rbac role skipped for no AccessRule matched
+    2018-07-26T22:25:41.183255Z debug rbac checking role productpage-viewer
+    2018-07-26T22:25:41.183281Z debug rbac matched AccessRule[0]
+    2018-07-26T22:25:41.183390Z debug rbac generated filter config: {policies:<key:"productpage-viewer" value:<permissions:<and_rules:<rules:<or_rules:<rules:<header:<name:":method" exact_match:"GET" > > > > > > principals:<and_ids:<ids:<any:true > > > > >  }
+    2018-07-26T22:25:41.184407Z info  rbac built filter config for productpage.default.svc.cluster.local
+    ... ...
+    {{< /text >}}
+
+    To make sure Pilot is handling the authorization policies correctly:
+
+    * Carefully check if there are any errors in the log.
+    * Check if there is a `built filter config for` message which means a filter config is generated
+      for the target service.
+
+    Taking the above output as an example:
+
+    * Pilot generated an empty config for `sleep.foo.svc.cluster.local` as there is no authorization
+      policies matched. This also means all requests sent to this service will be denied as Istio
+      authorization is deny by default.
+    * Pilot generated an config for `productpage.default.svc.cluster.local` that allows anyone to
+      access it with GET method.
+
+## Make sure policies are distributed to proxy
+
+The authorization policies are eventually distributed to and enforced in proxies. Run the following
+command to get the proxy config dump for the `productpage` service.
+
+> Note: The command used in this section assumes you have deployed [bookinfo application](/docs/examples/bookinfo/).
+You should replace `"-l app=productpage"` with your actual pod to get its config dump.
```

Comment:
pod-> pod label

Reply:
only the part after `-l` is pod label, I highlighted the whole `-l app-productpage` which should be replaced to pod name, or just pod to make it shorter.